packages/reporter-vue/cypress/support/index.ts

```diff
@@ -19,6 +20,10 @@ const tooltipOptions = {
   },
 }
 
+beforeEach(() => {
+  $('#__cy_root').css({ position: 'relative' })
```

Comment:
Could just use `Cypress.$` to save the import

Reply:
Typing issues.